spk/radarr/src/service-setup.sh

```diff
@@ -24,43 +25,43 @@ service_postinst ()
     echo "Set update required"
     # Make Radarr do an update check on start to avoid possible Radarr
     # downgrade when synocommunity package is updated
-    touch "${RADARR_CONFIG_DIR}/update_required" 2>&1
+    touch ${RADARR_CONFIG_DIR}/update_required 2>&1
 
-    if [ "${SYNOPKG_DSM_VERSION_MAJOR}" -lt 7 ]; then
+    if [ ${SYNOPKG_DSM_VERSION_MAJOR} -lt 7 ]; then
```

Comment:
BTW, keeping `"<var>"` in if statement changes the error msg (although exit status remains the same).

Using `"<var>"`:
```
./toto.sh: 3: [: Illegal number: 
```
without `""`:
```
./toto.sh: 3: [: -lt: unexpected operator
```

Reply:
BTW `SYNOPKG_DSM_VERSION_MAJOR` will never be empty, since we have a safeguard at the beginning of all generated service-setup files in the packages.

```
### Generic variables and functions
### -------------------------------

if [ -z "${SYNOPKG_PKGNAME}" ] || [ -z "${SYNOPKG_DSM_VERSION_MAJOR}" ]; then
  echo "Error: Environment variables are not set." 1>&2;
  echo "Please run me using synopkg instead. Example: \"synopkg start [packagename]\"" 1>&2;
  exit 1
fi
```

Reply:
and quoting in other palces will depend on whether DSM installation directories might ever contain spaces....